pages/agent/v3/windows.md.erb

```diff
@@ -22,3 +22,7 @@ See the [Agent SSH Keys](/docs/agent/v3/ssh-keys) documentation for more details
 ## Configuration
 
 See the [configuration documentation](/docs/agent/v3/configuration) for an explanation of each configuration setting.
+
+## Git for Windows
+
+While the agent will work without git installed, you will require [Git for Windows](https://gitforwindows.org/) to interact with git or ssh. In addition to providing git and ssh, Git for Windows includes Bash for Windows and a variety of standard linux tools which are frequently required for [plugins](plugins).
```

Comment:
Link here needs to be `/docs/agent/v3/plugins` and `Git` and `SSH` should have the correct casing.